portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -887,6 +887,63 @@ definition {
 			successPageTitle = "You have already responded.");
 	}
 
+	@description = "Verify that with the option 'Limit to one submission per user' activated and the form is on the page, the user cannot send more than one submission."
+	@priority = "4"
+	test CheckCantSendMoreThanOneSubmitPerUserOnPages {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
+
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Test Form");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.gotoSubmissionsTab();
+
+		Form.selectOptionSubmissions(checkboxOption = "Limit to One Submission per User");
+
+		Button.clickDone();
+
+		Form.publishForm();
+
+		ProductMenu.gotoPortlet(
```

Comment:
Instead of creating a Page and adding a Forms Portlet, what do you think about using FormsAdminNavigator.gotoPublishedForm(); to navigate directly to the Form and submitting from there. Then we would use less steps to this test.

Reply:
I need to go to the page to test sending the form from the page